### PR TITLE
Fixes #10: Count assigned PRs and reviews unique

### LINKED ISSUE #10: Being assigned and requested review on the same PR counts twice
When I am assigned to a PR and also the review is requested from me, i see 2 waiting PRs. It would be nice if it was grouped by PR so i see only 1.

### PR DESCRIPTION
**Pull request by sihu**
No longer double-counted PRs

### CODE REVIEW

File: extension.js [hunk 1]
```diff
@@ -56,8 +54,14 @@ function init() {
...
-function countPullRequests(issues) {
-    return issues.total_count;
+function getHtmlUrls(issues) {
+    return issues.items.map(function(review) { return review.pull_request.html_url; });
+}
+
+function getUniquePullRequestUrls(reviewsResponse, assigneesResponse) {
+    var reviews = getHtmlUrls(reviewsResponse);
+    var assignees = getHtmlUrls(assigneesResponse);
+    return reviews.concat(assignees).filter(function(v, i, a){ return a.indexOf(v) === i});
```
Comment (coorasse) [inline extension.js]: please rename the variables to something meaningful

File: extension.js [hunk 2]
```diff
@@ -31,11 +29,11 @@ init();
...
 function openCurrentURL() {
     if (tokenOk) {
-        if (totalCounter === 1) {
-            chrome.tabs.create({'url': pullRequestURL});
+        if (pullRequestUrls.length === 1) {
+            chrome.tabs.create({'url': pullRequestUrls[0]});
         }
         else {
-            if (totalCounter === 0 || assigneesCounter > 0) {
+            if (pullRequestUrls.length === 0 || pullRequestUrls.length !== reviewsCounter) {
```
Comment (coorasse) [inline extension.js]: can you explain me the second condition?

Comment (sihu) [reply extension.js]: if the pullRequestUrl length is not equal the length of reviews, that means, that there must be assigned pull requests, but i extracted it now!

File: extension.js [hunk 3]
```diff
@@ -30,12 +28,16 @@ setInterval(updateCounter, pollInterval * (60 * 1000));
 init();
 
 function openCurrentURL() {
+    function hasAssignedPullRequests() {
```
Comment (sihu) [inline extension.js]: since i cant make a private function i put it into this function. hope you are ok with that?

Comment (coorasse) [reply extension.js]: doesn't matter.

**Comment by coorasse** [comment]
I need to remember to deploy this new version. I'll make a note for myself

**Pull request merged**